scripts/base/frameworks/supervisor/main.zeek

```diff
@@ -42,6 +41,13 @@ function Supervisor::node(): Supervisor::NodeConfig
 
 event zeek_init() &priority=10
 	{
+	if ( Supervisor::is_supervisor() && SupervisorControl::enable_listen )
+		{
+		Broker::listen(Broker::default_listen_address,
```

Comment:
Looks good, one more nit: maybe this could move into a Broker-side function now, `Broker::listen_default()` or so.

Reply:
Nice one, yeah.